src/main/java/bg/codexio/springframework/data/jpa/requery/resolver/FilterJsonArgumentResolver.java

```diff
@@ -122,26 +120,24 @@ private Specification<Object> noFilterSpecification() {
      * recursively builds a composite {@link Specification} based on the
      * logical operations and groupings defined within the filter request.
      *
-     * @param complexFilterJson The JSON string containing the complex filter
-     *                          criteria.
-     * @param genericType       The entity class type on which the filter
-     *                          will be applied.
+     * @param complexFilter The parsed JSON string containing the complex filter
```

Comment:
Probably `The adapted {@link FilterGroupRequest} from the request`, as it might not only be JSON string - some protocols in the future may include XML or other proprietary formats.